### DISCUSSION TITLE
Accessing has many on user

### DISCUSSION THREAD

**Discussion by Keoghan** [statamic-rad-pack/runway#195] [Help]
Hi, 

Thanks for this package, it's a super addition to the world of Statamic.

I'm having trouble accessing a HasMany relationship on a user record. 

The relations are all in the DB, and I have the relationship set up in the \App\Models\User. I suspect the issue is that the instance of the User in the control panel is not an \App\Models\User but something else?

Any thoughts on how to work with it?

Thanks, Keoghan

**Comment by duncanmcclean** [comment]
Hey! I'm not quite sure why it's not working, sorry! 

I've opened a bug report (#196) which I'll take a look at when I have some time!
